src/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './index.css';
+import 'bootstrap/dist/css/bootstrap.css';
+
+const initialState = [];
+export default initialState;
 
```

Comment:
Are sure that the Redux store is not updated with the data from the API? Because I think in our project: the API data is stored in the Redux store and the web app works well. You can test it again.
We forgot to remove these useless 2 lines from the code, but the Redux store is updated and it doesn't affect the process of the application.